### PR TITLE
[Media] Misc updates on WebXR, MSE v2, Media Timed Events, WebRTC

### PR DESCRIPTION
**Pull request by tidoust**
This commit updates the contents of the Media roadmap to address points raised in https://github.com/w3c/media-and-entertainment/issues/7

Notably:
- Media control: mention WebXR instead of WebVR. Complete note to also talk about XR input devices.
- Media Distribution and Media Processing: mention incubation of codec switching
- Media Distribution and Media Processing: update MSE description to better highlight adaptive streaming
- Media Orchestration: mention the Media Timed Events spec
- Media Distribution: update part on WebRTC to mention WebRTC distribution
- Media Distribution: mention discussion on push and multicast in "Features not covered by on-going work" section

### CODE REVIEW

**Comment by tidoust** [comment]
@chrisn, could you have a look at proposed updates?

File: data/mse-v2.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      "url": "https://www.w3.org/community/wicg/",
+      "label": "Web Platform Incubator Community Group"
+    }
+  ],
+  "features": {
+    "codec-switching": {
+      "title": "Codec switching",
+      "impl": {
+        "chromestatus": 5719220952236032
```
Comment (chrisn) [inline data/mse-v2.json]: Here's the Firefox bug for this feature: https://bugzilla.mozilla.org/show_bug.cgi?id=1481166

File: media/distribution.html [hunk 2]
```diff
@@ -12,19 +12,22 @@ <h1>Media Distribution</h1>
...
+        <p data-feature="Adaptive streaming">To ensure the user experience of streamed media remains resilient in the face of changing network conditions, media providers make use of adaptive streaming, where content gets split into chunks at different quality levels, and where the client requests chunks at the appropriate quality level based on networking conditions during distribution. Several technologies are available to satisfy this approach (such as MPEG DASH) but Web browsers may not offer native support for them. The <a data-featureid="mse">Media Source Extensions</a> (MSE) specification enables Web application developers to create libraries that will consume these different adaptive streaming formats and protocols.</p>
```
Comment (chrisn) [inline media/distribution.html]: A minor comment, suggest "during playback" instead of "during distribution"?

File: media/processing.html [hunk 3]
```diff
@@ -22,14 +22,15 @@ <h2>Exploratory work</h2>
...
         </div>
+        <div data-feature="Adaptive streaming">
+          <p>Some scenarios require the ability to switch to a different codec during media playback. For instance, this need may arise on program boundaries when watching linear content. Also, typical ads are of a very different nature than the media stream that they need to be inserted into. The first version of the Media Source Extensions (MSE) specification does not address such heterogeneous scenarios, forcing content providers to use complex workarounds, e.g. using two <code>&lt;video&gt;</code> elements overlaid one top of the other, with no guarantees that transition between streams will be smooth. The Web Platform Incubator Working Group is incubating a <a data-featureid="mse-v2/codec-switching">codec switching feature for MSE</a>, to be integrated in a second version of MSE.</p>
```
Comment (chrisn) [inline media/processing.html]: Suggest "... will be smooth, or transcoding and re-packaging the ad content to be consistent with the program content"

**Comment by chrisn** [comment]
Thanks! I'll update the issue in the M&E repo.

**Pull request merged**